README.md

```diff
@@ -1,14 +1,35 @@
 # HyP3-lib
 
-## Common library for HyP3 plugins
+## CURRENTLY UNDER CONSTRUCTION!!!
+HyP3-lib is currently undergoing a major redesign and many utilities that are no longer used by ASF have been removed between version 2 and version 3. In addition, we plan to migrate some remaining functionality to other projects and to significantly refactor what remains. For this reason, **if you are using HyP3-lib we strongly recommend pinning to major versions at this time**. We further recommend not using this library in production until its version 4 release.
 
-### Please note that the use of `hyp3lib` version 2 or less is **_NOT_** recommended. 
+The use of HyP3-lib version 2 or less is **_NOT_** recommended.
```

Comment:
```suggestion
```

I don't think we need this since it's covered by the last line of the above alert